run_codenarc.py

```diff
@@ -92,7 +92,7 @@ def __init__(self, report_file: str) -> None:
 
 def _build_classpath(args: argparse.Namespace) -> str:
     """Construct the classpath to use for running CodeNarc."""
-    codenarc_version = _codenarc_version(args.codenarc_version, args.groovy4)
+    codenarc_version = _codenarc_version(args.codenarc_version, is_groovy4=args.groovy4)
```

Comment:
Wouldn't it be better to do:

```suggestion
    codenarc_version = _codenarc_version(version=args.codenarc_version, is_groovy4=args.groovy4)
```

...and thus eliminate the `*` from the signature of `_codenarc_version below`?